### ISSUE TITLE
about training parameters

### ISSUE THREAD

**Issue by HONGJUNRL** [xuxy09/QVI#2]
Thanks for your great job, I have some questions about the training parameters
In the paper, learning rate decay factor is set to be 0.1, and the loss weight is set to be 1 and 0.005 
but in this code,  learning rate decay factor is set to be 0.3, and the loss weight is set to be 204 and 0.005
Also, in the paper, you first train 200 epochs with flownet fixed and finetune the whole system for another 40 epochs, but in this code, you train 250 epochs with flownet fixed
